jmh-benchmarks/src/main/java/org/apache/kafka/jmh/fetcher/ReplicaFetcherThreadBenchmark.java

```diff
@@ -174,8 +173,14 @@ public int sizeInBytes() {
                     return null;
                 }
             };
-            initialFetched.put(tp, new FetchResponse.PartitionData<>(Errors.NONE, 0, 0, 0,
-                    new LinkedList<>(), fetched));
+            initialFetched.put(tp, new FetchResponseData.PartitionData()
+                    .setPartitionIndex(tp.partition())
+                    .setErrorCode(Errors.NONE.code())
+                    .setHighWatermark(0)
+                    .setLastStableOffset(0)
+                    .setLogStartOffset(0)
```

Comment:
We can remove some redundant setters?

Reply:
I will remove redundant setter `setHighWatermark(0)`. Other values are different from default value in KRPC.